lib/random-util.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const RANDOM_POOL_SIZE = 8192;
+const RANDOM_POOL_REFRESH = 1024;
+
+let randomPool = Buffer.alloc(RANDOM_POOL_SIZE);
```

Comment:
We don't need to .alloc this - as we use the buffer from the callback. Just set randomPoolIdx to RANDOM_POOL_SIZE to prevent "use-before-set"